### PR TITLE
Added the Printrbot Plus Metal Machine to machine definitions

### PR DESCRIPTION
**Pull request by swood333**

### CODE REVIEW

**Comment by nallath** [comment]
Please check the output of the CI checks. There are some issues with your profile.

**Comment by nallath** [comment]
@swood333 Could you please check if the suggested changes work?

File: resources/extruders/printrbot_plus_extruder_0.def.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "version": 2,
+    "name": "Extruder 1",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "printrbot_plus",
```
Comment (konskarm) [inline resources/extruders/printrbot_plus_extruder_0.def.json:6]: This should be "printrbot_metal_plus", same as the machine name

**Comment by nallath** [comment]
I'm closing this pull request due to inactivity. Feel free to create another one (or re-open this once) once the issues with it have been resolved.

**Pull request closed without merging**